runtimes/common/src/lib.rs

```diff
@@ -196,3 +202,16 @@ where
 		})
 	}
 }
+
+pub struct DustAndFeeHandler<T>(sp_std::marker::PhantomData<T>);
+
+impl<T> OnUnbalanced<CreditOf<T>> for DustAndFeeHandler<T>
+where
+	T: pallet_balances::Config,
+	T: pallet_treasury::Config,
+{
+	fn on_nonzero_unbalanced(amount: CreditOf<T>) {
+		let treasury_account_id = pallet_treasury::Pallet::<T>::account_id();
+		let _ = pallet_balances::Pallet::<T>::resolve(&treasury_account_id, amount);
```

Comment:
I think you can get rid of the initial `let _ =`.

Reply:
`resolve `is returning a `Result` which has to be handled. For better debugging, In [f1efce4](https://github.com/KILTprotocol/kilt-node/pull/527/commits/f1efce41b8166eb4cf4c4b721ac73e332e846550) I added a log message.